openvasd/openvasd.c

```diff
@@ -123,14 +123,11 @@ openvasd_connector_new (void)
 {
   openvasd_connector_t connector;
   connector = g_malloc0 (sizeof (struct openvasd_connector));
-  if (!connector)
-    return NULL;
-
   return connector;
 }
 
 /** @brief Build a openvasd connector
- *  @Description Recieve option name and value to build the openvasd connector
+ *  Receive option name and value to build the openvasd connector
```

Comment:
There needs to be a blank line before the description, else it is added to the brief.

Could you also clear out the doxygen warnings coming from openvasd/*. Eg missing param in openvas_build_scan_config_json.  Command is `make -B doc`, needs this:

```
diff --git doc/Doxyfile.in doc/Doxyfile.in
index d58e5aca..624b6f31 100644
--- doc/Doxyfile.in
+++ doc/Doxyfile.in
@@ -755,6 +755,7 @@ WARN_LOGFILE           =
 
 INPUT                  = @CMAKE_SOURCE_DIR@/base \
                          @CMAKE_SOURCE_DIR@/gmp \
+                         @CMAKE_SOURCE_DIR@/openvasd \
                          @CMAKE_SOURCE_DIR@/osp \
                          @CMAKE_SOURCE_DIR@/util
 ```